packages/nc-gui-v2/components/dlg/AirtableImport.vue

```diff
@@ -336,7 +336,8 @@ onBeforeUnmount(() => {
           </div>
         </a-card>
         <div class="flex justify-center items-center">
-          <a-button v-if="showGoToDashboardButton" class="mt-4" size="large" @click="dialogShow = false">Go to Dashboard</a-button
+          <a-button v-if="showGoToDashboardButton" class="mt-4" size="large" @click="dialogShow = false"
+            >Go to Dashboard</a-button
           >
```

Comment:
Broken formatting
`>` shouldn't be by themselves (even though eslint accepts this, it looks bad)

Reply:
Caused by `npm run lint`. May fix it later.